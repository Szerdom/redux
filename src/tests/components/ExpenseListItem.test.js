import React from 'react';
import { shallow } from 'enzyme';
import ExpenseListItem from '../../components/ExpenseListItem';
import expenses from '../fixture/expenses';

test('shoud render ExpenseListItem', () => {
    const wrapper = shallow(<ExpenseListItem expense={expenses[1]}/>);
    expect(wrapper).toMatchSnapshot();
});