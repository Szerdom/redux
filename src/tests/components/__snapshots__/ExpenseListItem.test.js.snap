// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shoud render ExpenseListItem 1`] = `
<div>
  <NavLink
    activeClassName="active"
    aria-current="page"
    to="/edit/undefined"
  >
    <h3 />
  </NavLink>
  <p>
     - 
  </p>
</div>
`;
